### DISCUSSION TITLE
CSS box model

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#58] [Q&A] [answered]
What are the components of the CSS box model?

**Comment by Bunny77K9** [comment, marked as answer]
The CSS box model is used to represent an entity that encloses all of the HTML content into a box or a button element.

There are four components:

    Border: Denotes the padding and content around the border
    Content: Refers to the actual content to be displayed
    Margin: Refers to the top layer of the box element
    Padding: Defines the empty space around the element
